src/components/grid-view/GridView.scss

```diff
@@ -2,49 +2,45 @@
 @import '../../styles/variables';
 
 .bdl-GridView {
-    padding: 0 5px;
-
-    &.bdl-GridView--columns-1 .bdl-GridView-slot {
-        $columns1prct: 100%;
-
-        flex-basis: $columns1prct;
-        max-width: $columns1prct;
-    }
-
-    &.bdl-GridView--columns-3 .bdl-GridView-slot {
-        $columns3prct: calc(33.3333% - 10px); /* 100%/3 - 10px */
-
-        flex-basis: $columns3prct;
-        max-width: $columns3prct;
+    .ReactVirtualized__Table__rowColumn:first-child {
+        margin: 0;
     }
 
-    &.bdl-GridView--columns-5 .bdl-GridView-slot {
-        $columns5prct: calc(20% - 10px); /* 100%/5 - 10px */
-
-        flex-basis: $columns5prct;
-        max-width: $columns5prct;
+    & .bce-btn-more-options {
+        padding: 4px 10px;
     }
+}
 
-    .ReactVirtualized__Table__rowColumn:first-of-type {
-        margin: 0;
-    }
+.bdl-GridView-body {
+    outline: none;
+    padding: 5px;
```

Comment:
Putting the padding here (as opposed to putting it on .bdl-GridView) does work as expected.